uw-frame-components/portal/messages/controllers.js

```diff
@@ -575,7 +576,9 @@ define(['angular'], function(angular) {
           if (actionType == addToHome) {
             var slash = url.lastIndexOf('/') + 1;
             var fName = url.substr(slash);
-            PortalAddToHomeService.addToHome(fName);
+            $rootScope.addPortletToHome(fName);
+            actionButton.label = 'Added To Home';
```

Comment:
Prefer sentence case. A sentence, even.

Reply:
The widget exists
You got this already
Desire leads to suffering
Why you wanna add a thing what you currently posses

-What should the sentence say?